cmd/helm/root.go

```diff
@@ -68,10 +70,17 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 
 	// Add the registry client based on settings
 	// TODO: Move this elsewhere (first, settings.Init() must move)
+	// TODO: handle errors
+	credentialsFile := filepath.Join(settings.Home.Registry(), registry.CredentialsFileBasename)
+	client, _ := auth.NewClient(credentialsFile)
+	resolver, _ := client.Resolver(context.Background())
```

Comment:
It seems currently oras resolver only has default options. Do we plan to have a way to configure different resolver options, like PlainHTTP (for example running a test registry on HTTP but not locahost)?

Reply:
I'd be hesitant to encourage any use of http from the onset. This can of course come later if needed

Reply:
I agree with you. The point is not the PlainHTTP support. Just point out the possibility that user may want to extend the default resolver, which resolves the host name of the server, schema, and auth.

I understand it's not necessary for now; just brought it up so we think about possibility of extension (probably plugin model).